src/bicep/modules/policyAssignment.bicep

```diff
@@ -0,0 +1,42 @@
+targetScope = 'resourceGroup'
+
+param builtInAssignment string = ''
+param logAnalyticsWorkspaceName string
+param workspaceResourceGroupName string
+param policyAssignmentPrefix string = 'MLZ -'
```

Comment:
if this is needed, let's allow a user to specify the `policyAssignmentPrefix` from the root `mlz.bicep` module -- or better yet, is this needed? Am I wrong to assume that `${resourceGroup().name} ${modifiedAssignment}` is specific enough? 

The bigger motivation is to remove the hardcoded `mlz` artifacts and always allow a user to specify names of things.

Reply:
Unlike other resources policy assignme3nts may show in a larger list without any other way of determining they come from or are associated with MLZ, they can modify the parameter if needed but it is here and set to a default for using this as a separate deployment. Its should be a 'if they don't set it it thing', I'm okay in removing it just my thinking on it.

Reply:
Ah I understand now. I'm for removing it until it's a problem we need to solve. I think so long as we give them the breadcrumb to the resource group it originates from that's a good enough start.

Reply:
Removed